hack/lib/protoc.sh

```diff
@@ -71,20 +71,20 @@ function kube::protoc::format() {
   local package=${1}
 
   # Update boilerplate for the generated file.
-  echo "$(cat hack/boilerplate/boilerplate.generatego.txt ${package}/api.pb.go)" > ${package}/api.pb.go
+  cat hack/boilerplate/boilerplate.generatego.txt "${package}/api.pb.go" > tmpfile && mv tmpfile "${package}/api.pb.go"
```

Comment:
change reason:
```
In hack/lib/protoc.sh line 74:
  echo "$(cat hack/boilerplate/boilerplate.generatego.txt "${package}/api.pb.go")" > "${package}/api.pb.go"
       ^-- SC2005: Useless echo? Instead of 'echo $(cmd)', just use 'cmd'.
```

```
In hack/lib/protoc.sh line 74:
  cat hack/boilerplate/boilerplate.generatego.txt "${package}/api.pb.go" > "${package}/api.pb.go"
                                                  ^-- SC2094: Make sure not to read and write the same file in the same pipeline.
                                                                           ^-- SC2094: Make sure not to read and write the same file in the same pipeline.
```

ref: https://github.com/koalaman/shellcheck/wiki/SC2094